### DISCUSSION TITLE
HmIP RTV - Programs

### DISCUSSION THREAD

**Discussion by ChristophCaina** [SukramJ/aiohomematic#1537] [Q&A] [answered]
Hi all,
In the CCU, a Thermostat can be set into "Automatic", "Manual", "Boost" and "Holiday" Mode.
In addition to that, you can select between 3 Week-Programs.

![image](https://github.com/danielperna84/hahomematic/assets/26391061/196c5a25-2fde-45e1-aa4b-f8da7cf70205)

In HomeAssistant, while using the default Thermostat Card, I can select between:

"Automatic", "Heating Mode" and "Off" - as well as one of the following Programs:
- Boost
- None
- Weekprogram 1, 2 or 3

![image](https://github.com/danielperna84/hahomematic/assets/26391061/a8813a73-a0d3-456a-ac4a-efc909066057)

![image](https://github.com/danielperna84/hahomematic/assets/26391061/3e32c120-a0a7-42f3-96e9-0a99919abb9a)

Is there a way how to enable the other modes that are available in the CCU to be available in the Thermostat Card / In HomeAssistant as well?

Also, I noticed, that if I change the Program in HomeAssistant, it seems not to be changed in Homematic...
![image](https://github.com/danielperna84/hahomematic/assets/26391061/cc3a4756-bec0-4820-a116-4ce146ce9791)

**Comment by SukramJ** [comment, marked as answer]
Also fehlt nur `Holiday`?

Da man ein Datum setzten muss, geht das nur als Service.

Siehe:
https://github.com/danielperna84/custom_homematic#homematicip_localenable_away_mode_by_duration
https://github.com/danielperna84/custom_homematic#homematicip_localenable_away_mode_by_calendar
https://github.com/danielperna84/custom_homematic#homematicip_localdisable_away_mode
